pkgs/test/texlive/default.nix

```
@@ -1,6 +1,31 @@
 { lib, stdenv, buildEnv, runCommand, fetchurl, file, texlive, writeShellScript, writeText }:
 
-{
+rec {
+
+  mkTeXTest = lib.makeOverridable (
+    { name
+    , format
+    , tex
```

```suggestion
    , text
```
Maybe `text` (as in `pkgs.writeTextFile` ) is a better name for this argument as `tex` could  me misinterpreted more easliy.